docs/developer/order-expiration.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
```

Comment:
Camel case here, doesn't say anything here - I mean you don't link it to Channel type etc so it won't give you any hint. 
I would change it to a normal name like "Order expiration has been introduced in XX"